.github/workflows/parallel.yml

```diff
@@ -84,11 +84,11 @@ jobs:
         restore-keys: |
           mvn-${{ inputs.java-version }}-package-${{ matrix.profile }}-
```

Comment:
It does not seem to me that this one needs to change, having said that I'm rather new to github workflows so any advice on this would be appreciated.